Modules/_sqlite/connection.c

```diff
@@ -281,6 +296,11 @@ static PyObject *
 pysqlite_connection_cursor_impl(pysqlite_Connection *self, PyObject *factory)
 /*[clinic end generated code: output=562432a9e6af2aa1 input=4127345aa091b650]*/
 {
+    if (self == NULL) {
+        return NULL;
+    }
+        
+    static char *kwlist[] = {"factory", NULL};
```

Comment:
This is not needed if `self->initialized` is only set if `pysqlite_connection_init` was successful. Also, it looks like the rebase left some artifacts :)